docs/contributing/test-suite.rst

```diff
@@ -7,10 +7,36 @@
 **********
 Test suite
 **********
```

Comment:
minor: a blank line here would be useful, that's typically done below headers in .rst files.